addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -531,7 +536,7 @@ msgstr ""
 
 #: /xml/VideoOSD.xml
 msgctxt "#31115"
-msgid "Toggle subtitle"
+msgid "Subtitles streams"
```

Comment:
I consider this change risky as you never know whether other skins also use this string for whatever. With this change you might "break" them. I would introduce a new string instead of changing the existing string.

Reply:
replied above